src/components/ingredients/IngredientModalTwo.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  Heading,
+  NumberDecrementStepper,
+  NumberIncrementStepper,
+  NumberInput,
+  NumberInputField,
+  NumberInputStepper,
+} from "@chakra-ui/react";
+
+const IngredientModalTwo = () => {
```

Comment:
Tohle chapu, ze jeste dotahnes, ale super zacatek 👍 Libi se mi, ze jsi to takhle elegantne rovnou prehodila do vlastniho "modalu".